Vungle/VungleBannerCustomEvent.m

```diff
@@ -61,10 +64,10 @@ - (void)requestAdWithSize:(CGSize)size adapterInfo:(NSDictionary *)info adMarkup
     
     if (@available(iOS 10.0, *)) {
         self.timeOutTimer = [NSTimer scheduledTimerWithTimeInterval:BANNER_TIMEOUT_INTERVAL repeats:NO block:^(NSTimer * _Nonnull timer) {
-        if (!self.isAdCached) {
-            [[VungleRouter sharedRouter] clearDelegateForRequestingBanner];
-        }
-    }];
+            if (!self.isAdCached) {
+                [[VungleRouter sharedRouter] clearDelegateForRequestingBanner];
+            }
+        }];
```

Comment:
extra spaces.

Reply:
I don't think these are extra spaces. This indents the if-statement to the block.

See the original (https://github.com/Vungle/mopub-ios-mediation/blob/6.9.x_pre_release/Vungle/VungleBannerCustomEvent.m#L63)